src/makeInfinityAdder.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
 /***
+ *
  * @return {function}
  */
+
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (arg) => {
```

Comment:
The parameter `arg` could be given a default value of 0. This would simplify the check for `undefined` and the resetting of `result`.